bin/Socetnet2ISIS.py

```diff
@@ -0,0 +1,49 @@
+import os
+import numpy as np
+
+# Reads a .atf file and outputs all of the
+# .ipf, .gpf, .sup, .prj, and path to locate the
+# .apf file (should be the same as all others)
+def read_atf(atf_file):
+    with open(atf_file) as f:
+
+        files = []
+        ipf = []
+        sup = []
+        files_dict = []
+
+        # Grabs every PRJ, GPF, SUP, and IPF image from the ATF file
+        for line in f:
+            if line[-4:-1] == 'prj' or line[-4:-1] == 'gpf' or line[-4:-1] == 'sup' or line[-4:-1] == 'ipf' or line[-4:-1] == 'atf':
```

Comment:
@tthatcher95 @acpaquette Something to consider do a quick refactor on: https://docs.python.org/3/library/os.path.html#os.path.splitext